katportalclient/test/test_client.py

```diff
@@ -1424,12 +1541,10 @@ def mock_fetch(url, method="GET", body=None):
         start_ok = starts_with is None or url.startswith(starts_with)
         end_ok = ends_with is None or url.endswith(ends_with)
         contains_ok = contains is None or contains in url
-
         if start_ok and end_ok and contains_ok:
             body_buffer = StringIO.StringIO(valid_response)
         else:
             body_buffer = StringIO.StringIO(invalid_response)
-
         # optionally send raw message from test websocket server
```

Comment:
As mentioned above, this whole part about publishing websocket messages and client states is no longer required.  katstore64 is much simpler to use.